test/torchaudio_unittest/functional/autograd_impl.py

```diff
@@ -43,14 +44,6 @@ def test_lfilter_a(self):
         a.requires_grad = True
         self.assert_grad(F.lfilter, (x, a, b), enable_all_grad=False)
 
-    def test_lfilter_b(self):
```

Comment:
What was the reason to remove this test again?

Reply:
Thanks for pointing it out. After bb1a792 we should add back the test.